pkgs/top-level/aliases.nix

```diff
@@ -383,6 +383,7 @@ mapAliases ({
   gmailieer = lieer; # Added 2020-04-19
   gmic_krita_qt = gmic-qt-krita; # Added 2019-09-07
   gmvault = throw "gmvault has been removed because it is unmaintained, mostly broken, and insecure"; # Added 2021-03-08
+  gnash = throw "gnash has been removed; broken and abandoned upstream."; # added 2022-02-06
```

Comment:
1. It's not broken:
```
λ git revert 79cb125a6d27f3846cd2e60ae71926de95bae78b
Auto-merging pkgs/top-level/all-packages.nix
Auto-merging pkgs/top-level/aliases.nix
[master 79247e69dc6] Revert "gnash: remove"
 4 files changed, 193 insertions(+), 1 deletion(-)
 create mode 100644 pkgs/misc/gnash/0001-fix-build-with-ffmepg-4.patch
 create mode 100644 pkgs/misc/gnash/default.nix
λ nix build -f. gnash
[1 built, 300 copied (1193.3 MiB), 250.5 MiB DL]
```

2. It's not broken because several people are making patches, despite being unmmaintained upstream.

3. You shouldn't just remove packages without pinging their maintainers.

Reply:
The switch of ffmpeg default was reverted later, so I assume that's why it isn't broken now.

Reply:
It's also working fine:
![image](https://user-images.githubusercontent.com/2817565/153711784-876e81b2-b137-4dbd-9687-bfd747f36f55.png)

Reply:
> 1. It's not broken:
> 
> 

Like @vcunat said, it was broken with ffmpeg 5. That was reverted, so of course it's not broken anymore.

> ```
> λ git revert 79cb125a6d27f3846cd2e60ae71926de95bae78b
> Auto-merging pkgs/top-level/all-packages.nix
> Auto-merging pkgs/top-level/aliases.nix
> [master 79247e69dc6] Revert "gnash: remove"
>  4 files changed, 193 insertions(+), 1 deletion(-)
>  create mode 100644 pkgs/misc/gnash/0001-fix-build-with-ffmepg-4.patch
>  create mode 100644 pkgs/misc/gnash/default.nix
> λ nix build -f. gnash
> [1 built, 300 copied (1193.3 MiB), 250.5 MiB DL]
> ```
> 
>     2. It's not broken because several people are making patches, despite being unmaintained upstream.
> 

Where? The most recent patch on the `gnash-dev` mailing list is a fix for building with ffmpeg 4 - look who the author is... Also, I checked upstream before removing it.

>     3. You shouldn't just remove packages without pinging their maintainers.

This is your only valid criticism, I didn't realise ofborg wouldn't ping maintainers when their package is removed. I'll be sure to check next time.